fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -1566,11 +1565,9 @@ struct TenantManagementWorkload : TestWorkload {
 				ASSERT(!specialKeysUseInvalidTuple);
 				ASSERT(!assignToDifferentCluster);
 				Versionstamp currentVersionstamp = wait(getLastTenantModification(self, operationType));
-				if (configurationChanging) {
+				if (tenantGroupChanging) {
```

Comment:
We need the versionstamp check to be done if we are updating any configuration. Right now, `tenantGroupChanging` == `configurationChanging`, but that may not always be the case.

Reply:
Make sense. Will update.

Reply:
Maybe I'm misinterpreting this, but doesn't this still have the same issue? If we update line 1539 in the future, this line is checking the wrong configuration flag because it does not encode the possibility of `tenantGroupChanging || assignToDifferentCluster.` Is there any reason not to use `configurationChanging` all the way throughout instead of temporarily changing what we call it?

Reply:
I think this block is specific to tenant-group changing, that's why it is inside the `if (tenantGroupChanging)`

Reply:
Ok, I see what's going on now. The original comment was on an old revision and shows up on a different line number when I view it on the latest commit. I can see that this comment was referring to what is now line 1559. Sorry for the confusion